web/client/reducers/mapInfo.js

```diff
@@ -288,6 +289,12 @@ function mapInfo(state = initState, action) {
             filterNameList: action.filterNameList || null
         });
     }
+    case UPDATE_MAPINFO_STATE_WITH_CLICK_POINT: {
```

Comment:
could you not reuse FEATURE_INFO_CLICK ?

Reply:
More precisely I think you should intercept the CLICK_ON_MAP action as well as the actual filter does. Click on map action should contain all the info, included the modifiers, so you don't need anything else, i think.

Reply:
Map info is dedicated to feature info tool. In this case the click on map is handled by the other tool, so the mapinfo actions/apics should not be included at all

Reply:
> @MV88 about this [comment](https://github.com/geosolutions-it/MapStore2/pull/6121#pullrequestreview-519826589) I can't seem to see the numbers info anymore. Maybe something was changed in the FeatureEditor code?

i mean the text at the bottom left when you have multiple rows selected
![image](https://user-images.githubusercontent.com/11991428/97992493-07c40280-1de3-11eb-9b74-ffec3b98b2f8.png)

Reply:
Alright. I had a small screen size with console open so I wasn't seeing it. I have fixed the count issue.